menu_item.cpp

```diff
@@ -194,6 +201,12 @@ void MenuItemTest::CursorUp() { parent->CursorUpAndActivate(); }
 
 void MenuItemTest::CursorDown() { parent->CursorDownAndActivate(); }
 
+void MenuItemTest::OnX() {
```

Comment:
This is an interesting usability thing. If you're inside of a test, would you expect hitting X to run all of the tests starting from a test that may be further up in the list? Or would you expect it to just run the tests after the current test?

I think maybe I'd leave this one out and require X to be hit on the suite, hitting inside a test would have no effect (see the comment below on the `OnX` naming).